be/src/olap/tablet.cpp

```diff
@@ -76,7 +76,7 @@ OLAPStatus Tablet::init_once() {
     for (int ser = 0; ser < _tablet_meta.rowset_size(); ++ser) {
         const RowsetMeta* rs_meta = _tablet_meta.get_rs_meta(ser);
         Version version = rs_meta->version();
-        Rowset* rowset = new Rowset(rs_meta);
+        std::shared_ptr rowset(new Rowset(rs_meta));
         _version_rowset_map[version] = rowset;
```

Comment:
std::shared_ptr<Rowset> ?

It is better to define RowsetSharedPtr instead of using std::shared_ptr<Rowset>